datasets/eu/esma_sanctions/crawler.py

```diff
@@ -0,0 +1,75 @@
+from lxml import html
+
+from zavod import Context, helpers
+
+
+def get_json(context: Context, batch_size: int):
+    params = {
+        "q": "*",
+        "rows": batch_size,
+        "start": 0,
+        "wt": "json",
+    }
+    cont = True
+    while cont:
+        resp = context.fetch_json(context.data_url, params=params)
+        for doc in resp["response"]["docs"]:
+            yield doc
+        params["start"] += batch_size
+        cont = resp["response"]["numFound"] >= params["start"]
+
+
+def parse_name(name_markup: str) -> str:
+    """
+    Remove markup from name if present.
+    """
+    if name_markup is None:
+        return None
+    name = html.fromstring(name_markup).text_content().strip()
+    return name
+
+
+def crawl(context: Context) -> None:
+    for row in get_json(context, 1000):
+        if (
+            row.get("sn_entityName") is None
```

Comment:
One alternative way to do this is to just throw all of these into `context.make_id` together, and see if the result of the whole thing is `None` :)

Reply:
Makes sense, changed it